src/typedefs.ts

```diff
@@ -1,16 +1,25 @@
-//@ts-nocheck
+// https://www.typescriptlang.org/docs/handbook/utility-types.html
+
 interface NominalTag<T> {
   'nominalTag': T;
 }
 
 type Nominal<Type, Tag> = NominalTag<Tag> & Type;
 
-const enum Degree {}
-const enum Radian {}
+// eslint-disable-next-line no-unused-vars
+const enum Degree { }
+// eslint-disable-next-line no-unused-vars
+const enum Radian { }
 
 export type TDegree = Nominal<number, Degree>;
 export type TRadian = Nominal<number, Radian>;
 
+export type TSize = {
+  width: number;
+  height: number;
+}
```

Comment:
I added something similar in the utils,
I wonder if TSize is an interface more than a type, meaning that anything that has a width and height can be used in something that accepts a size

Reply:
How do you see the difference between interface and type?